.github/workflows/node.js.yml

```diff
@@ -6,8 +6,8 @@ name: Node.js CI
 on:
   push:
     branches:
-      - trunk
-      - feature/718-tezori3-poc
+      
+      - feature/718-tezori3-poc-ticket2
```

Comment:
You shouldn't change this Github action. It's not part of this ticket and it will break our continuous integration process.